Tests/Client/Unit Tests/Client_SilentMessageTests.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  Client_SilentMessageTests.swift
+//  StreamChatClientTests
+//
+//  Created by Bahadir Oncel on 12.05.2020.
+//  Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import XCTest
+@testable import StreamChatClient
+
+class Client_SilentMessageTests: XCTestCase {
+    
+    private static let config = Client.Config(apiKey: "client_silentMessageTests")
+    private let config = Client_SilentMessageTests.config
+    private var client: Client {
+        Client(config: config)
+    }
+
+    class override func setUp() {
+        Client.configureShared(config)
```

Comment:
Shouldn't this cause `assertionFailure` because you configure the shared instance twice? Here and in `Channel_SilentMessageTests `?

Reply:
`class override func setUp()` is `class` func so it only runs once!

Reply:
No way around this. You will probably have to move it outside of the test case class and treat it as a proper singleton.

Reply:
`class func setUp` is called once per test suite (all tests within the test case), but you have two test suites where you call it.

Reply:
OK, fixed!